src/applications/simple-forms/20-10207/config/constants.js

```diff
@@ -283,13 +283,205 @@ export const FINANCIAL_HARDSHIP_DESCRIPTION = Object.freeze(
     </ul>
     <va-additional-info
       trigger="What kind of evidence can I submit?"
-      data-testid="hardshipAdditionalInfo"
+      data-testid="financialHardshipAdditionalInfo"
     >
       <ul>
         <li>Copy of an eviction notice or statement of foreclosure</li>
         <li>Copy of notices of past-due utility bills</li>
         <li>Copy of collection notices from creditors</li>
       </ul>
     </va-additional-info>
+    <p />
```

Comment:
same, looks like you could do some refactoring here if you like

Reply:
Agreed.  I'll try refactoring for DRYer in Part 5.  Thanks!